config/traject.rb

```diff
@@ -255,7 +255,7 @@
 
 ## Subject browse facet
 to_field 'subject_browse_facet', process_subject_browse_facet(
-  standard_fields: '650|*0|abcdgvxyz:650|*1|abcdgvxyz:650|*3|abcdgvxyz:650|*3|abcdgvxyz',
+  standard_fields: '600|*0|abcd:610|*0|abcd:611|*0|abcd:630|*0|avxyz:647|*0|avxyz:650|*0|abcdgvxyz:650|*1|abcdgvxyz:650|*2|abcdgvxyz:650|*3|abcdgvxyz:651|*0|agvxyz',
```

Comment:
Do we need a test to make sure we don't get any regressions? Doesn't have to be much... use the existing ones, or write one specifically for this change.

Reply:
I thought about it as well but was not sure looking at the existing ones but I can def. add sth 👍🏻

Reply:
specs added